src/signals/incident/components/form/ContainerSelect/components/ContainerEditList/index.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useCallback } from 'react';
+import type { FunctionComponent } from 'react';
+import styled from 'styled-components';
+
+import { themeSpacing } from '@amsterdam/asc-ui';
+import { Close } from '@amsterdam/asc-assets';
+
+import Button from 'components/Button';
+import type { FeatureType, Item } from '../../types';
+
+const DEFAULT_ICON_SIZE = 40;
+const REMOVE_BUTTON_SIZE = 32;
```

Comment:
I think these constants are not adding to much, till now in the code the icon sizes are not specified as constants. I would keep that style for this kind of constants.